src/startWorkers.ts

```diff
@@ -1,5 +1,6 @@
 import dotenv from 'dotenv'
 dotenv.config() // keep this line first in file
+process.env.WORKER = 'true'
```

Comment:
In this context, I think it makes sense to use process.env to make the discord bots in a certain way.

The env variable makes sense too, since it tells all code it's running in a worker context.